packages/web-core/src/services/ConnectService.ts

```diff
@@ -121,16 +121,16 @@ export class ConnectService {
   }
 
   async loginInit(abortController: AbortController): Promise<Result<ConnectLoginInitData, CorbadoError>> {
-    const existingProcess = ConnectProcess.loadFromStorage(this.#projectId);
-    if (existingProcess?.isValid()) {
-      log.debug('process exists, preparing api clients');
-      this.#setApisV2(existingProcess);
-    }
+    // const existingProcess = ConnectProcess.loadFromStorage(this.#projectId);
```

Comment:
this effectively makes sure that we get a new process every time when a login is initiated

Reply:
yes makes sense.